Readme.md

```diff
@@ -184,6 +184,92 @@ form.submit({
 });
 ```
 
+### Methods
+
+#####_void_ append( **String** _field_, **Mixed** _value_ [, **Mixed** _options_] )
```

Comment:
Should here add space between `#####` and `_void_`? Otherwise it will not be compiled as a title. 

![image](https://user-images.githubusercontent.com/4194287/60067664-148c0d00-973e-11e9-9967-10b90b950920.png)

Reply:
Nice catch! phpstorm does render it, didn't check on github b4 submitting :)
fixed it